agents/PCS-Large-Exhange-Rate-Update/src/agent.ts

```diff
@@ -0,0 +1,60 @@
+import { Finding, HandleTransaction, TransactionEvent, getEthersProvider } from "forta-agent";
+import { BigNumber } from "ethers";
+import DataFetcher from "./data.fetcher";
+import { createFinding } from "./utils";
+import {
+  SWAP_EVENT,
+  ER_THRESHOLD,
+  PANCAKE_FACTORY_ADDRESS,
+  INIT_CODE_PAIR_HASH,
+  STKBNB_POOL_ADDRESS,
+  STKBNB_TOKEN_ADDRESS,
+  WBNB_TOKEN_ADDRESS,
+} from "./constants";
+
+export const provideBotHandler = (erThreshold: Number, fetcher: DataFetcher): HandleTransaction => {
```

Comment:
Can't we directly use `ER_THRESHOLD` from constants instead of passing here?
I can't see the use of `erThreshold` parameter as it always uses constant everywhere or you can use custom `TEST_ER_THRESHOLD` in test cases.

Reply:
Isnt this more modular?